README.md

```diff
@@ -30,71 +30,20 @@ For more context, starts with this [stackoverflow thread](http://stackoverflow.c
 
 In order to use this token acquisition method, you need to configure a service principal. Please follow [this walkthrough](https://azure.microsoft.com/en-us/documentation/articles/resource-group-create-service-principal-portal/).
 
-See the [sample](./sample/client_credentials_sample.py).
-```python
-import adal
-
-context = adal.AuthenticationContext('https://login.microsoftonline.com/ABCDEFGH-1234-1234-1234-ABCDEFGHIJKL')
-RESOURCE = '00000002-0000-0000-c000-000000000000' #AAD graph resource
-token = context.acquire_token_with_client_credentials(
-    RESOURCE,
-    "http://PythonSDK", 
-    "Key-Configured-In-Portal")
-```
+See the [sample](sample/client_credentials_sample.py#L46-L53).
```

Comment:
If for maintainability, line range is also a concern though as it could get out of date easily. Maybe you just provide the file link and then call out the mark, like comment text, where the pertinent code starts